### ISSUE TITLE
Doesn't work with the lastest openai package or API url

### ISSUE THREAD

**Issue by liyupeng111** [kjappelbaum/gptchem#22]
It seems that 'from openai.cli import FineTune as FineTuneCli' was only compatible with versions on and before 0.28.1 of the openai package.
But if installing the 0.28.1 version, the fine-tune API url is wrong.

Has anyone tested the "getting started" code lately?
Do you have the same issue?

**Comment by kjappelbaum** [comment]
yes, the code here assumes the old version and has not been migrated

**Comment by FrostyPB1023** [comment]
Hi,

Thank you for clarifying that the current code assumes compatibility with older versions of the OpenAI package. I wanted to ask if there are any plans to migrate the code to support newer versions of the OpenAI API (>= 0.29.0)?

If there's an updated branch or workaround available, could you please share it? I'm particularly interested in running the fine-tune functionality with the latest API.
